options/hooks/useFetchKeywordSearchData.js

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useEffect, useState } from "react";
+
+import { SERVER_URL } from "../constants/constants";
+
+const useFetchKeywordSearchList = (setCrawledResult, bookmarkList) => {
+  const [keyword, setKeyword] = useState("");
+  const [searchResultCount, setSearchResultCount] = useState(false);
```

Comment:
```suggestion
  const [hasSearchResult, setHasSearchResult] = useState(false);
```
변수명이 헷갈려서 리뷰 남깁니다. 검색을 시작했다면 true로 바뀌는 값이라고 해석이 됩니다.
그런데 뒤에 Count가 붙어 있다보니 검색 횟수를 저장하는 변수라고 해석이 되서 다음과 같이 컨벤션명을 제안드립니다.

Reply:
boolean값일 때의 컨벤션이 누락되었네요. 좋은 제안 및 확인감사합니다.
참조하여 변수 명 교체하겠습니다.